src/views/Requests/index.tsx

```diff
@@ -84,25 +74,20 @@ const Requests: React.FC<{ isLoading?: boolean }> = (props) => {
         };
 
         communities();
-    }, [myCountrySelected, review, itemOffset, asPath]);
+    }, [review, itemOffset, asPath]);
 
-    //  Tabs (countries and review) numbers
+    //  Filter countries and tabs numbers
     useEffect(() => {
         const tabsNumber = async () => {
             try {
                 setNumbersLoading(true)
-                
+
                 //  Number for tabs
-                const allCountries = await getReviewsByCountry({
-                    status: "pending",
-                }).unwrap()
-                const otherCountries = await getReviewsByCountry({
-                    excludeCountry: userCountry,
-                    status: "pending"
-                }).unwrap()
+                const reviewsByCountryCount = await getReviewsByCountry({
```

Comment:
Can't this data be fetched by your newly created hook useCommunitiesReviewsByCountry('pending', fetcher)?